src/ServiceStack.OrmLite/OrmLiteWriteExpressionsApi.cs

```diff
@@ -246,7 +246,7 @@ public static int Delete<T>(this IDbConnection dbConn, Expression<Func<T, bool>>
         ///   db.Delete&lt;Person&gt;(q.Where(p => p.Age == 27));
         ///   DELETE FROM "Person" WHERE ("Age" = 27)
         /// </summary>
-        public static int Delete<T>(this IDbConnection dbConn, SqlExpression<T> where)
+        public static int Delete<T>(this IDbConnection dbConn, SqlExpression<T> where, Action<IDbCommand> commandFilter = null)
         {
             return dbConn.Exec(dbCmd => dbCmd.Delete(where));
```

Comment:
shouldn't this be: ... dbCmd.Delete(where, commandFilter)); ?

Reply:
Yep it should be, thx for catching it! I'll resolve it on my side.

Reply:
Ops